utils/serviceRequest.js

```diff
@@ -44,19 +48,45 @@ class ABServiceRequest extends ServiceCote {
          domainRequesters[domain] = new cote.Requester({
             name: `${this.req.serviceKey} > requester > ${domain}`,
             key: domain,
+            // https://github.com/dashersw/cote/blob/master/src/components/requester.js#L16
+            timeout: REQUEST_TIMEOUT,
          });
       }
-      domainRequesters[domain].send(paramStack, (err, results) => {
-         if (this.req.performance) {
-            this.req.performance.measure(key, key);
-         }
-         clearInterval(timeOutID);
-         if (err) {
-            err._serviceRequest = key;
-            err._params = paramStack;
-         }
-         cb(err, results);
-      });
+
+      const sendRequest = () => {
+         countRequest += 1;
+
+         domainRequesters[domain].send(paramStack, (err, results) => {
+            if (this.req.performance) {
+               this.req.performance.measure(key, key);
+            }
+            // clearInterval(timeOutID);
+            if (err) {
+               // https://github.com/dashersw/cote/blob/master/src/components/requester.js#L132
+               if (err.message === "Request timed out.") {
+                  // Retry .send
+                  if (countRequest < ATTEMPT_REQUEST_MAXIMUM) {
+                     this.req.log(
+                        `...It is ${countRequest} time to re-request(${key})`
```

Comment:
Just reword the message a bit:
`... timeout waiting for request (${key}), retrying ${count request}/${ATTEMPT_REQUEST_MAXIMUM}`